src/components/bds-switch/bds-switch.tsx

```diff
@@ -0,0 +1,126 @@
+import { Component, h, Prop, State, Method, Event, EventEmitter, Watch } from '@stencil/core';
+export type SwitchSize = 'tall' | 'standard' | 'short';
+
+let switchIds = 0;
+
+@Component({
+  tag: 'bds-switch',
+  styleUrl: 'bds-switch.scss',
+  shadow: true,
+})
+export class Switch {
+  private nativeInput?: HTMLInputElement;
+
+  @State() switchId?: string;
+
+  @Prop() refer!: string;
+
+  /**
+   * Size. Entered as one of the size. Can be one of:
+   * 'tall', 'standard', 'short';
+   * An evolution for the component will be to apply responsiveness (make it dynamic)
```

Comment:
as these comments are generated as public documentation, I think it's not necessary to keep this comment (line 21)

Reply:
This comment was a request from Tathi. Balbo suggested adding this way, would you have a better place to put it?

Reply:
Resolved